e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -200,23 +200,154 @@ test.describe('Example Imagery', () => {
+    await bgImageLocator.hover();
+    const imageMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+    expect(imageMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+    expect(imageMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+    // Center the mouse pointer
+    await page.mouse.move(imageCenterX, imageCenterY);
+
+    // Pan Imagery Hints
+    console.log(process.platform);
+    const expectedAltText = process.platform === 'linux' ? 'Ctrl+Alt drag to pan' : 'Alt drag to pan';
+    const imageryHintsText = await page.locator('.c-imagery__hints').innerText();
+    expect(expectedAltText).toEqual(imageryHintsText);
+
+    // Click next image button
+    await nextImageButton.click();
+
+    // Click fixed timespan button
+    await page.locator('.c-button__label >> text=Fixed Timespan').click();
+
+    // Click local clock 
+    await page.locator('.icon-clock >> text=Local Clock').click();
+
+    // Zoom in on next image
+    await bgImageLocator.hover();
+    await page.mouse.wheel(0, deltaYStep * 2);
+    const zoomedNextBoundingBox = await bgImageLocator.boundingBox();
+    const imageNextCenterX = zoomedNextBoundingBox.x + zoomedNextBoundingBox.width / 2;
+    const imageNextCenterY = zoomedNextBoundingBox.y + zoomedNextBoundingBox.height / 2;
+
+    // Verify real-time/fixed-time imagery will pause
+    const pausePlayButton = await page.locator('.c-button.pause-play');
+    expect(pausePlayButton).toHaveClass('c-button icon-pause pause-play is-paused');
+
+    // Wait for zoom animation to finish
+    await bgImageLocator.hover();
+    const imageNextMouseZoomedIn = await page.locator(backgroundImageSelector).boundingBox();
+    expect(imageNextMouseZoomedIn.height).toBeGreaterThan(originalImageDimensions.height);
+    expect(imageNextMouseZoomedIn.width).toBeGreaterThan(originalImageDimensions.width);
+
+     // Click previous image button
+     await previousImageButton.click();
+
+     // Verify previous image
+     await expect(selectedImage).toBeVisible();
+
+    // Verify real-time/fixed-time imagery will pause
+    expect(pausePlayButton).toHaveClass('c-button icon-pause pause-play is-paused');
+
+    // Wait 20ms to verify no new image has come in
+    // sleep time expects milliseconds
+    function sleep (time) {
+        return new Promise((resolve) => setTimeout(resolve, time));
+    }
+    
+    // Sleep 21ms
+    sleep(21);
+
+    // Verify next image is Disabled
+    await nextImageButton.isDisabled();
+
+    //Get background-image url from background-image css prop
+    const backgroundImage = await page.locator('.c-imagery__main-image__background-image');
+    let backgroundImageUrl = await backgroundImage.evaluate((el) => {
+        return window.getComputedStyle(el).getPropertyValue('background-image').match(/url\(([^)]+)\)/)[1];
+    });
+    backgroundImageUrl1 = backgroundImageUrl.slice(1, -1); //forgive me, padre
```

Comment:
ha! My comment survived :) 

I had to hack together a url matcher with regex and couldn't figure out how to get the url between the parentheses AND the double quotes

Reply:
well it lives on!